unlock-app/src/config.js

```diff
@@ -144,6 +144,26 @@ export default function configure(
     blockTime = 8000
   }
 
+  if (env === 'dev-kovan') {
+    // In dev-kovan, the network can only be Kovan
+    isRequiredNetwork = networkId => networkId === 42
+    chainExplorerUrlBuilders.etherScan = address =>
+      `https://kovan.etherscan.io/address/${address}`
+    requiredNetworkId = 42
+    paywallUrl = 'https://'
+    supportedProviders = ['Metamask', 'Opera']
+    services['storage'] = { host: runtimeConfig.locksmithHost }
+    services['wedlocks'] = { host: runtimeConfig.wedlocksUri }
+    paywallUrl = runtimeConfig.paywallUrl
```

Comment:
Duplicate definition here, not sure if that's in the other blocks, but if it is we should look into it to avoid any latent bugs!

Reply:
Yeah, I pretty much copy&pasted from the block above... If you have any changes you'd like to see around this let me know.